src/components/SubmitConfirmModal/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Modal } from '@edx/paragon';
+
+
+const SubmitConfirmModal = ({
+  onSubmit,
+  ...others
+}) => (
+  <Modal
+    {...others}
+    title="Submit for Review?"
+    body="You will not be able to make edits while the course is in review, which can take up to 48 hours. Confirm your edits are complete."
```

Comment:
Bigger message. More information.

After this point, you can no longer update entitlements (or mode/price or whatever). Maybe something about submitting runs is still allowed?

Maybe talk to Carol about messaging here

Reply:
Talked to Carol. She isn't so worried about emphasizing that submitting runs is still OK. But likes the idea of warning about mode changes.

But she thought the modal was maybe too late for that and asked it to be like, above the field. I'll look at that change.

Reply:
Okay, I'm cool with whatever. Just thought we should add in a little something.